.githooks/pre-commit

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -e -o pipefail
+
+if ! command -v golangci-lint &> /dev/null; then
+    echo "golangci-lint is not installed"
+    exit 1
+fi
+
+exec golangci-lint run "$@"
```

Comment:
`golangci-lint --build-tags=race run`

Reply:
Do you want to check all files all the time? Even if commiter even not touch it?

Reply:
Golangci-lint caches files. But `--build-tags=race` is about the internal RR's flag for the unsafe buffer check and fast string<->[]byte conversion: https://github.com/roadrunner-server/sdk/blob/master/utils/race_checker.go

Reply:
I mean, add this flag to the command, like: `exec golangci-lint --build-tags=race run "$@"`

Reply:
@rustatian  got it, updated